integration-tests/mockApis/auth.js

```diff
@@ -124,7 +124,7 @@ const stubUserMe = ({
     body: { username, firstName, lastName, name, activeCaseLoadId },
   })
 
-const stubUserMeRoles = (roles) => getFor({ urlPattern: '/auth/api/user/me/roles', body: roles })
+const stubUserMeRoles = (roles) => getFor({ urlPattern: '/users/me/roles', body: roles })
```

Comment:
Should this stub be in mockApis/manageusers.js

Reply:
Is this used anywhere?

Reply:
Moved to manageusers.js